README.adoc

```diff
@@ -234,7 +234,7 @@ endif::[]
 
 ifndef::cloud-hosted[]
 
-You can retrieve the information about the resource usage of any number of system services at the `GET /query/systemLoad/{hostnames}` endpoint. When the `hostnames` are specified to `system-java8,system-java11`. The response is similar to the following example:
+You can retrieve the information about the resource usage of any number of system services at the `GET /query/systemLoad/{hostnames}` endpoint. When the `hostnames` are specified to `system-java8,system-java11`, the response is similar to the following example:
```

Comment:
See previous comment regarding specified vs set, and the phrasing of the sentence that leads in to the example. There may be other instances so can you check.